text/0174-metrics-api.md

```diff
@@ -0,0 +1,147 @@
+# Metrics API
+
+This endpoint is currently experimental.
+
+This means that it can break at any time between two minor versions as long as it is not stabilized.
```

Comment:
```suggestion
This means it can break between two minor versions at any time as long as it is not stabilized. It also means it could be removed and never be stabilized.
```